libs/gltfio/src/Animator.cpp

```diff
@@ -453,4 +337,139 @@ const char* Animator::getAnimationName(size_t animationIndex) const {
     return mImpl->animations[animationIndex].name.c_str();
 }
 
+
+void AnimatorImpl::addChannels(const NodeMap& nodeMap, const cgltf_animation& srcAnim,
+        Animation& dst) {
+    cgltf_animation_channel* srcChannels = srcAnim.channels;
+    cgltf_animation_sampler* srcSamplers = srcAnim.samplers;
+    const Sampler* samplers = dst.samplers.data();
+    for (cgltf_size j = 0, nchans = srcAnim.channels_count; j < nchans; ++j) {
+        const cgltf_animation_channel& srcChannel = srcChannels[j];
+        auto iter = nodeMap.find(srcChannel.target_node);
+        if (iter == nodeMap.end()) {
+            slog.w << "No scene root contains node ";
+            if (srcChannel.target_node->name) {
+                slog.w << "'" << srcChannel.target_node->name << "' ";
```

Comment:
We probably don't wait logs in release builds. Also, the condition for the log needs a `unlikely`.

Reply:
Fixed, added a macro to help me omit these in release build.  There are lots of non-catastrophic errors that can occur when loading a gltf so I should probably do this elsewhere in the library too.